client/signup/config/test/index.js

```diff
@@ -8,10 +8,22 @@ import isEmpty from 'lodash/isEmpty';
 /**
  * Internal dependencies
  */
-import flows from '../flows';
-import steps from '../steps';
+import useFilesystemMocks from 'test/helpers/use-filesystem-mocks';
+import useMockery from 'test/helpers/use-mockery';
+
+describe( 'index', () => {
+	let flows, steps;
+
+	useFilesystemMocks( __dirname );
```

Comment:
i wonder if useFilesystemMocks should be a thing we call from within useMockery? I think we're setting up a double-enable / double-cleanup here. It'll "work" but it's kinda fragile.